### PR TITLE
Update Chapter template with template for Personal Stories

### LINKED ISSUE #1473: Create a series of template to facilitate contributions
### Summary

* As the book has evolved, it's content has also started to develop in modular size that makes the template outdated
* We need to create new templates to assist our contributors create their contents

### What needs to be done?

These templates should be crowdsourced with the help of @paulowoicho:

- [x] Template for a landing page of the chapter
- [x] Template for a small subchapter 
- [x] Case studies for reproducibility
- [x] Case studies for communication
- [x] Case studies for collaboration
- [x] Case studies for ethical research
- [x] Case studies for project design
- [ ] Personal interview or a personal story (generally can be added as a last subchapter of a chapter)

A few more things to think about:

- [x] When creating sections in a template, add prompts to help authors brainstorm what they should think about their content
- [x] How they can cross link subchapters if any
- [x] Maybe add the short consistency checklist as comment at the end of these templates to ensure that they can review their contribution before sending a PR for review
- [x] Take time to consider if specific guide will require some specific prompts or one template will fit all guides

### Who can help?

* Anyone who is interested in helping develop this

---

### PR DESCRIPTION
**Pull request by paulowoicho**
### Summary

Partially Fixes #1473

I add a template for writing personal stories to the new chapter template in the Turing Way and fix broken links + typos in the earlier chapter template.

### List of changes proposed in this PR (pull-request)

- [x] Template for personal stories
- [x] Fix broken links and typos
- [x] Added a note about the order of chapter files

### What should a reviewer concentrate their feedback on?

- [ ] Are the prompts for the chapter template adequate to support contributors looking to write a personal story?
- [ ] Everything looks ok?

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [x] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

File: book/templates/chapter-template/README.md [hunk 1]
```diff
@@ -49,25 +52,37 @@ When adding your chapter to _The Turing Way_ book, your landing page should live
 
 ### `chapter-content.md`:
 
-Inside the `chapter-content` folder, you will find the [`chapter-content.md`](templates\chapter-template\chapter-content\chapter-content.md) file where you can write about the individual topics that make up your chapter.
+Inside the `chapter-content` folder, you will find the [`chapter-content.md`](templates/chapter-template/chapter-content/hapter-content.md) file where you can write about the individual topics that make up your chapter.
```
Comment (paulowoicho) [inline book/templates/chapter-template/README.md:55]:
```suggestion
Inside the `chapter-content` folder, you will find the [`chapter-content.md`](templates/chapter-template/chapter-content/chapter-content.md) file where you can write about the individual topics that make up your chapter.
```

File: book/templates/chapter-template/chapter-content/chapter-personal-stories.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## `Subject Name`: `Story Caption`
+
+> The title of the personal story should start with the name of the subject followed by a short caption that captures the overall message of the story.
+> For example, the [personal story](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-stories.html) in the Research Data Management chapter is aptly titled: `Susanna-Assunta Sansone: From FAIR Co-Author to FAIR Doer`
+
+> Regardless of how you choose to format the personal story, ensure that it contains the elements itemised in checklist below. Feel free to use images, external resouces, or code blocks to tell a compelling story.
+> Above all, centre the personal story around how readers are expected to benefit from the subject's experiences
+
+> ### Checklist
```
Comment (malvikasharan) [inline book/templates/chapter-template/chapter-content/chapter-personal-stories.md:21]: The markdown formatting won't work. So maybe we can drop that at the moment.

**Comment by malvikasharan** [comment]
@paulowoicho I think the README page for the chapter content should also be published in the community handbook right after the consistency chapter. What do you think?

**Comment by paulowoicho** [comment]
> @paulowoicho I think the README page for the chapter content should also be published in the community handbook right after the consistency chapter. What do you think?

I think that would be helpful, and it'll be easier for contributors to find the rest of the template as well. I can think of how to revise and merge this README with that of the case study template before making a PR

File: book/templates/chapter-template/chapter-content/chapter-personal-stories.md (hunk 2 above)
Comment (paulowoicho) [reply book/templates/chapter-template/chapter-content/chapter-personal-stories.md]: Thanks @malvikasharan. I have updated the PR to remove the checklist formatting

**Comment by malvikasharan** [comment]
Perfect. We can create one chapter with the README file that you have written, and add links to all the templates with an overview of what readers can expect. Great work putting all of these together.

**Review by malvikasharan — APPROVED** [review]

File: book/templates/chapter-template/chapter-content/chapter-personal-stories.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  - Avoiding HTML
+  - Consecutive headers
+  - Labels and cross referencing
+  - Using images
+  - Latin abbreviations
+  - References and citations
+  - Title casing
+  - Matching headers with reference in table of content
+
+-->
```
Comment (malvikasharan) [inline book/templates/chapter-template/chapter-content/chapter-personal-stories.md:54]:
```suggestion

## BEFORE YOU GO

> - Use this template to create a personal story.
> - Refrain from writing very long content as readers may be unwilling to read them. Rather, you should split long content into smaller subchapters if necessary.
> - Have a look at the Style Guide and the Maintaining Consistency chapters to ensure that you have followed the relevant recommendations on
>   - Avoiding HTML
>   - Consecutive headers
>   - Labels and cross referencing
>   - Using images
>   - Latin abbreviations
>   - References and citations
>   - Title casing
>   - Matching headers with reference in table of content
```

Comment (malvikasharan) [reply book/templates/chapter-template/chapter-content/chapter-personal-stories.md:54]: I think it's ok to have these visible in the rendered file. A disclaimer can be added on the top that asks authors to remove the quoted block.

**Review by malvikasharan — APPROVED** [review]

File: book/templates/chapter-template/chapter-content/chapter-personal-stories.md (hunk 3 above)
Comment (malvikasharan) [reply book/templates/chapter-template/chapter-content/chapter-personal-stories.md:54]: In consistency with the chapter template, I am adding this change.

**Comment by malvikasharan** [comment]
This will have a conflict with the chapter template - so to avoid that, first, merge the chapter template. Rebase this branch to sync with the main branch (`git pull origin master`). And then merge this branch.

**Pull request merged**